src/core/ticker/Ticker.js

```diff
@@ -321,7 +374,30 @@ export default class Ticker
             this.deltaTime = elapsedMS * settings.TARGET_FPMS * this.speed;
 
             // Invoke listeners added to internal emitter
-            this._emitter.emit(TICK, this.deltaTime);
+            let listener = this._head.next;
+
+            while (listener)
+            {
+                const previous = listener.previous;
+
+                listener.emit(this.deltaTime);
+
+                // Once listener or destroyed itself
+                if (listener.destroyed)
+                {
+                    listener = previous.next;
+                }
+                else
+                {
+                    // Go to the next listener
+                    listener = listener.next;
+                }
+            }
```

Comment:
I might be reading it wrong, but wouldn't that while loop and `destroy()` result in emitting `previous` before and after the destroyed listener?